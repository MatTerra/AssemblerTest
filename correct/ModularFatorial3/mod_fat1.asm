mod_main: begin
mod_fat: extern
n: extern
public aux
section data
aux: space
section text
input n
load n
jmp mod_fat
end
