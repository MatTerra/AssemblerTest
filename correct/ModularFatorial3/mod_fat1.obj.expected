12 25 10 25 05 07 00 02 21 08 22 11 06 03 25 11 25 10 06 05 22 01 13 25 14 00 